Building bohrium without having LAPACKE headers gives an error

Trying to build bohrium without the LAPACKE headers gives the following error:

```
/home/rlj/thesis/bohrium/build/extmethods/lapack/gesvp.cpp:12:25: fatal error: lapacke.h: No such file or directory
compilation terminated.
extmethods/lapack/CMakeFiles/bh_lapack_gesvp.dir/build.make:66: recipe for target 'extmethods/lapack/CMakeFiles/bh_lapack_gesvp.dir/gesvp.cpp.o' failed
make[2]: *** [extmethods/lapack/CMakeFiles/bh_lapack_gesvp.dir/gesvp.cpp.o] Error 1
CMakeFiles/Makefile2:886: recipe for target 'extmethods/lapack/CMakeFiles/bh_lapack_gesvp.dir/all' failed
make[1]: *** [extmethods/lapack/CMakeFiles/bh_lapack_gesvp.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
```

--- a/cmake/Modules/FindLAPACKE.cmake
+++ b/cmake/Modules/FindLAPACKE.cmake
@@ -1,0 +1,23 @@
+# Find and link to LAPACKE.
+#
+# Variables defined::
+#
+#   LAPACKE_FOUND
+#   LAPACKE_LIBRARIES
+#   LAPACKE_INCLUDE_DIR
+
+include(FindPackageHandleStandardArgs)
+
+set(LAPACKE_FOUND FALSE)
+set(LAPACKE_LIBRARIES "NOTFOUND")
+set(LAPACKE_INCLUDE_DIR "NOTFOUND")
+
+if(APPLE)
+  find_library (LAPACKE_LIBRARIES NAMES Accelerate)
+  find_path (LAPACKE_INCLUDE_DIR Accelerate.h)
+else()
+  find_library(LAPACKE_LIBRARIES NAMES lapacke)
+  find_path(LAPACKE_INCLUDE_DIR lapacke.h)
+endif()
+
+find_package_handle_standard_args(Lapacke DEFAULT_MSG LAPACKE_LIBRARIES LAPACKE_INCLUDE_DIR)

--- a/extmethods/lapack/CMakeLists.txt
+++ b/extmethods/lapack/CMakeLists.txt
@@ -6,22 +6,22 @@ if(NOT EXT_LAPACK)
 endif()
 
 #External dependencies
-find_package(LAPACK)
-set_package_properties(LAPACK PROPERTIES DESCRIPTION "Linear Algebra PACKage" URL "www.netlib.org/lapack")
-set_package_properties(LAPACK PROPERTIES TYPE RECOMMENDED PURPOSE "Enables the LAPACK extended method")
+find_package(LAPACKE)
+set_package_properties(LAPACKE PROPERTIES DESCRIPTION "Linear Algebra PACKage" URL "www.netlib.org/lapack/lapacke.html")
+set_package_properties(LAPACKE PROPERTIES TYPE RECOMMENDED PURPOSE "Enables the LAPACK extended method")
 
 find_path(CLAPACK_INCLUDE_DIR NAMES clapack.h PATHS /usr/include/atlas)
 if(CLAPACK_INCLUDE_DIR)
   include_directories(${CLAPACK_INCLUDE_DIR})
 endif()
 
-if(LAPACK_FOUND)
+if(LAPACKE_FOUND)
     set(EXTMETHOD_DIR ${CMAKE_BINARY_DIR}/extmethods/lapack)
     file(MAKE_DIRECTORY ${EXTMETHOD_DIR})
 
     include_directories(${CMAKE_SOURCE_DIR}/include)
     include_directories(${CMAKE_BINARY_DIR}/include)
-    include_directories(${LAPACK_INCLUDES})
+    include_directories(${LAPACKE_INCLUDE_DIR})
 
     set(GEN_EXTMETHOD ${CMAKE_CURRENT_SOURCE_DIR}/../../core/codegen/gen_extmethod.py)
 
@@ -48,7 +48,7 @@ if(LAPACK_FOUND)
           target_link_libraries(bh_lapack_${DRIVER} "-framework Accelerate")
         else()
           target_link_libraries(bh_lapack_${DRIVER} "-llapacke -lblas")
-          target_link_libraries(bh_lapack_${DRIVER} ${LAPACK_LIBRARIES})
+          target_link_libraries(bh_lapack_${DRIVER} ${LAPACKE_LIBRARIES})
         endif()
 
         install(TARGETS bh_lapack_${DRIVER} DESTINATION ${LIBDIR} COMPONENT bohrium)
@@ -57,8 +57,4 @@ if(LAPACK_FOUND)
     endforeach(DRIVER)
 
     set(OPENMP_LIBS ${OPENMP_LIBS} ${LAPACK_DRIVER_LIBS} PARENT_SCOPE)
-else()
-    if (LAPACK_INCLUDES_PRE)
-        set_package_properties(LAPACK PROPERTIES TYPE RECOMMENDED PURPOSE "LAPACK was found, but not enabled. \n   Manually enable LAPACK with LAPACK_INCLUDES.")
-    endif()
 endif()

